dradis/dradis-ce
# Notifications per project

### Spec
Notifications appearing in different projects

### Proposed solution
Add a project_id column to notifications to make the association much easier to handle.

### How to test
1. Create 2 users.
2. Create 2 projects with some issues. Subscribe the users in an issue in each project.
3. Using one account, comment on a single issue in a project.
4. Using the other account, confirm that the other project has no notifications.
